src/app/modules/project/overview/overview.component.html

```diff
@@ -48,12 +37,23 @@ <h4>Sort Options</h4>
       <div class="filter-group">
         <h4>Pagination</h4>
         <label>Projects per page</label>
-        <select (ngModelChange)="onChangePaginationSelect($event)" [(ngModel)]="defaultPaginationOption.id" class="option-select">
-          <option *ngFor="let paginationOption of paginationDropDownOptions" [ngValue]="paginationOption.id" class="paginationDropDown">
+        <select (change)="onPaginationChange()" [formControl]="paginationOptionControl" class="option-select">
+          <option *ngFor="let paginationOption of paginationDropDownOptions" [ngValue]="paginationOption" class="paginationDropDown">
             {{ paginationOption.amountOnPage }}
           </option>
         </select>
       </div>
+      <div class="filter-group" *ngIf="displayTags()">
+        <h4>Categories</h4>
+        <div class="project-categories">
+          <ng-container *ngFor="let category of categories">
+            <p>
+              <input type="checkbox" [ngModel]="category.selected" [name]="category.name" (change)="onCategoryChange(category.id)">
+              <label [for]="category.name">{{category.name}}</label>
```

Comment:
I guess we could improve styling of this one. Maybe merge it and improve next sprint?